### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by johnalpha74**
Add contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,6 +53,312 @@ <h1 class="heading">Hey there. I’m Sean John.<br /> <span id="green-text">I’
...
+        </section>
+
+        <div class="section contact-section">
+            <div class="container contact">
+                <div class="contact-info">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </div>
+
+                <div class="contact-form">
+                    <form class="form-block" action="submit">
```
Comment (AnselemOdims) [inline index.html]: - [x] Please make sure the `form` as an action and a method attribute that integrates with the formspree service.

File: index.html [hunk 2]
```diff
@@ -53,6 +53,312 @@ <h1 class="heading">Hey there. I’m Sean John.<br /> <span id="green-text">I’
...
+                        <div class="input-grid">
+                            <input placeholder="Name" type="text">
+                            <input placeholder="Last Name" type="text">
```
Comment (AnselemOdims) [inline index.html]: - [x] Please kindly include a `maxlength` attribute to the firstname and lastname inputs and give it a value of 30
- [x] Please include the required attribute for the name input fields

File: index.html [hunk 3]
```diff
@@ -53,6 +53,312 @@ <h1 class="heading">Hey there. I’m Sean John.<br /> <span id="green-text">I’
...
+                        </div>
+                        <input placeholder="Email Address" type="email" name="email" id="email">
+                        <textarea placeholder="Message" name="textarea" id="textarea" cols="30" rows="6"></textarea>
```
Comment (AnselemOdims) [inline index.html]: - [x] Please include the required attribute for the email and textarea input fields

File: index.html [hunk 4]
```diff
@@ -53,6 +53,312 @@ <h1 class="heading">Hey there. I’m Sean John.<br /> <span id="green-text">I’
...
+                        <div class="button-group">
+                            <button class="see-project" type="submit">Get in touch</button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
```
Comment (AnselemOdims) [inline index.html:358]: - [x] Please ensure that you stick to the design guideline as much as possible
- [x] The design only has one input field for `fullname`. Please only use one name input
- [x] Please ensure that the heading text is aligned to the center like that of the original design
- [x] Please kindly include the icons and email that is below the form in the original design

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @johnalpha74 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 5]
```diff
@@ -345,13 +345,13 @@ <h3 class="about-title">About Me</h3>
                 </div>
 
                 <div class="contact-form">
-                    <form class="form-block" action="submit">
+                    <form class="form-block" action="https://formspree.io/f/xeqnpgqv" method="post">
                         <div class="input-grid">
-                            <input placeholder="Name" type="text">
-                            <input placeholder="Last Name" type="text">
+                            <input placeholder="Full name" type="text" maxlength="30" required>
+                            <!-- <input placeholder="Last Name" type="text" maxlength="30" required> -->
```
Comment (chrystalme) [inline index.html]: Kindly remove commented code from your code-base. To keep your code-base clean and looking professional.

Comment (chrystalme) [inline index.html]: Kindly remove commented code from your code-base. To keep your code-base clean and looking professional.

File: index.html [hunk 6]
```diff
@@ -345,13 +345,13 @@ <h3 class="about-title">About Me</h3>
                 </div>
 
                 <div class="contact-form">
-                    <form class="form-block" action="submit">
+                    <form class="form-block" action="https://formspree.io/f/xeqnpgqv" method="post">
```
Comment (chrystalme) [inline index.html]: Why not use `<section>` tag for the contact form section. This make your code SEO friendly and semantically correct.

**Review by chrystalme — CHANGES_REQUESTED** [review]
# Hello @johnalpha74, 
Almost there

Great work on making the changes 👏 👍🏽 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes ♻️ 

- [x] Your README.md file has a bug. Kindly fix it.

- [x] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly ensure that as partners you always have balanced commits in pair programming project 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**